File: app/src/main/java/com/example/neofr/habitcounter/model/Habit.java
package com.example.neofr.habitcounter.model;


import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity
public class Habit {
    @PrimaryKey(autoGenerate = true)
    public long id;
    public String name;
    public String description;

    public Habit() {
    }

    public Habit(String name, String desc) {
        this.name = name;
        this.description = desc;
    }
}

File: app/src/main/java/com/example/neofr/habitcounter/model/Habit.kt
package com.example.neofr.habitcounter.model

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
data class Habit(
    @PrimaryKey(autoGenerate = true) val id: Long,
    val name: String,
    val description: String
)
